### ISSUE TITLE
mkdir: add "--context" option

### ISSUE THREAD

**Issue by Ecordonnier** [uutils/coreutils#5682]
I didn't find an issue tracking this, so I'm creating one for tracking purpose.
As noted in https://github.com/uutils/coreutils/blob/6bcf4e2cece5815eb7352e74f64af14b13ad5a4f/src/uu/mkdir/src/mkdir.rs#L84 , the option "-Z" / "--context" of mkdir is not yet implemented.

man mkdir indicates:

>        --context[=CTX]
>               like -Z, or if CTX is specified then set the SELinux or SMACK security context to CTX

**Comment by cre4ture** [comment]
I was testing my local mkdir (Ubuntu 22.04.3 LTS) for this option and I got this warning:
`mkdir: warning: ignoring --context; it requires an SELinux/SMACK-enabled kernel`
I think its worth to give some hints which distribution/system can be used for testing this feature.

**Comment by sylvestre** [comment]
you have to install selinux on your ubuntu:
https://www.linode.com/docs/guides/how-to-install-selinux-on-ubuntu-22-04/

**Comment by Ecordonnier** [comment]
Yes,  I'm able to test on Ubuntu after installing selinux. I started working on the implementation (but my progress is slow since I'm a noob in rust).

**Comment by AjithPanneerselvam** [comment]
I can give it a go at this 👍  

I will assign this to myself now.

**Comment by cakebaker** [comment]
@AjithPanneerselvam sure, go ahead :)

**Comment by Ecordonnier** [comment]
Closing as duplicate of https://github.com/uutils/coreutils/issues/2404
